### PR TITLE
[WIP] DRM/KMS backend (needs cleanup !)

### PR DESCRIPTION
**Pull request by TSnake41**

### CODE REVIEW

File: src/raylib.h [hunk 1]
```diff
@@ -852,6 +852,14 @@ typedef enum {
 // Callbacks to be implemented by users
 typedef void (*TraceLogCallback)(int logType, const char *text, va_list args);
 
+// Filesystem override implemented by users.
```
Comment (raysan5) [inline src/raylib.h:855]: Probably this filesystem related code shouldn't be in this PR.

File: examples/Makefile [hunk 2]
```diff
@@ -31,8 +31,8 @@ RAYLIB_PATH        ?= ..
 
 # Define default options
 
-# One of PLATFORM_DESKTOP, PLATFORM_RPI, PLATFORM_ANDROID, PLATFORM_WEB
-PLATFORM           ?= PLATFORM_DESKTOP
+# One of PLATFORM_DESKTOP, PLATFORM_RPI, PLATFORM_ANDROID, PLATFORM_WEB, PLATFORM_DRM
+PLATFORM           ?= PLATFORM_DRM
```
Comment (raysan5) [inline examples/Makefile:35]: Default Makefile PLATFORM or BUILD_MODE shouldn't be changed.

File: src/rlgl.h [hunk 3]
```diff
@@ -675,7 +675,11 @@ RLAPI int GetPixelDataSize(int width, int height, int format);// Get pixel data
 
 #if defined(GRAPHICS_API_OPENGL_ES2)
     #define GL_GLEXT_PROTOTYPES
+
+    /* We are not using EGL with X11. */
```
Comment (raysan5) [inline src/rlgl.h:679]: raylib naming convention use `// One line comments`

File: src/raylib.h (hunk 1 above)
Comment (TSnake41) [reply src/raylib.h:855]: Yes, that's not excepted, it needs to be stripped out of this PR.

**Comment by raysan5** [comment]
Closing this PR in favour of https://github.com/raysan5/raylib/pull/1388

**Pull request closed without merging**